client/app/bundles/HelloWorld/components/NewResturantForm.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class NewResturantForm extends React.Component {
+
+    static propTypes = {
+        cuisines: PropTypes.array.isRequired
+    };
+
+    /*componentDidMount() {
+        console.log('mounted');
+        navigator.geolocation.getCurrentPosition((position) => {
+            console.log(position)
+        })
+    }*/
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            name: '',
+            cuisine: '',
+            tenbis: false,
+            address: '',
+            maxDeliveryTime: ''
+        }
+    };
+
+    newResturantSubmit = (e) => {
+        e.preventDefault();
+        $.ajax({
```

Comment:
You can use the native ES6 `fetch` function instead of jQuery's ajax for XHR requests